src/cljs/rems/administration/create_form.cljs

```diff
@@ -185,8 +185,10 @@
                    :handler (flash-message/default-success-handler
                              description
                              (fn [response]
-                               (dispatch! (str "#/administration/forms/" (or (::form-id db)
-                                                                             (:id response))))))
+                               (dispatch! (str "#/administration/forms/"
+                                               (if edit?
+                                                 (db ::form-id)
+                                                 (:id %))))))
```

Comment:
hmm, (:id response)?

Reply:
(oh, I think Esko changed the order of the arguments so that it would be (::form-id db) instead of (db ::form-id). )

Reply:
fixed, tested and amended. I'll push a separate commit for the reorder.

Reply:
Thanks!